spec/helpers/monitoring.ts

```diff
@@ -457,7 +458,10 @@ class Monitoring {
             var itemElem = this.getSpikedItem(item);
 
             browser.actions().mouseMove(itemElem).perform();
-            itemElem.element(by.className('icon-dots-vertical')).click();
+            var dotsElem = itemElem.element(by.className('icon-dots-vertical'));
```

Comment:
Use `data-test-id` for selectors.

Can you also update `executeContextMenuAction` function and use it here?